### PR TITLE
Avoid dupes

### PR DESCRIPTION
**Pull request by dadap**
Fix additional duplicated entries, and add a check to xml2json.py

### CODE REVIEW

File: mem-27-extra.xml [hunk 1]
```diff
@@ -7016,7 +7016,7 @@ The reason that the language is {bama Hol:n:nolink} while the country is {mIyama
       <column name="search_tags_zh_HK"></column>
       <column name="search_tags_pt"></column>
       <column name="search_tags_fi"></column>
-      <column name="source">[1] {KLI mailing list 2022.12.30:src}</column>
+      <column name="source">[1] {KLI mailing list 2022.12.30:src}, [2] {Toki Pona - The Language of Good:src}</column>
```
Comment (De7vID) [inline mem-27-extra.xml:7019]: The `part_of_speech` should also be corrected to `n:place,terran` above.

File: mem-03-D.xml [hunk 2]
```diff
@@ -11625,7 +11625,7 @@ The logo of Star Alliance, the largest airline alliance on Earth, is often paint
       <column name="definition_fi">keskipäivä</column>
       <column name="synonyms">{pemjep:n}</column>
       <column name="antonyms">{ramjep:n:1}</column>
-      <column name="see_also">{megh:n}, {po:n}, {pov:n}</column>
+      <column name="see_also">{megh:n}, {po:n:1}, {pov:n}</column>
```
Comment (De7vID) [inline mem-03-D.xml]: This should be `{po:n:1h}` to "hide" the number in the link. This is down (maybe not completely consistently) for homophones which clash with names. For example, see `paq:n`.

Actually, I've updated the links to `po:n` already, so if you merge with master, it should be fixed.

Comment (dadap) [reply mem-03-D.xml]: Yeah, this series already includes a merge commit: https://github.com/De7vID/klingon-assistant-data/pull/762/commits/93565bcda68ac9e8ffc4b95505d000e05f022cdc

I'll just rebase it and incorporate the tag fix from your other comment.

**Comment by dadap** [comment]
I pushed a rebased v2 series, eliminating the need for the merge commit to resolve the n:1 vs. n:1h issue.

**Pull request merged**